Source/WebCore/html/HTMLVideoElement.cpp

```diff
@@ -113,6 +115,47 @@ void HTMLVideoElement::didAttachRenderers()
     }
 }
 
+void HTMLVideoElement::acceleratedRenderingStateChanged()
+{
+    maybeSyncAcceleratedRenderingState();
+}
+
+bool HTMLVideoElement::supportsAcceleratedRendering() const
+{
+    return RefPtr { player() } && player()->supportsAcceleratedRendering();
+}
+
+void HTMLVideoElement::mediaPlayerRenderingModeChanged()
+{
+    ALWAYS_LOG(LOGIDENTIFIER);
+
+    // Kick off a fake recalcStyle that will update the compositing tree.
+    maybeSyncAcceleratedRenderingState();
+    invalidateStyleAndLayerComposition();
+}
+
+void HTMLVideoElement::maybeSyncAcceleratedRenderingState()
```

Comment:
"maybe" implies this does something lazily, but it really only does something if state changed, so you could drop the "maybe". 

"sync" begs the question "synchronize with what". The media player, I guess? So could this be `computeAcceleratedRenderingStateAndUpdateMediaPlayer()` ?